src/cli/chirp.cc

```diff
@@ -3,7 +3,7 @@
 #include <gflags/gflags.h>
 
 #include "cli_parser.h"
-
+using namespace std;
```

Comment:
i don't think you're supposed to use global namespaces

Reply:
Fixed, I used it for debugging and forget to delete